cmd/controller/controller-runtime-main.go

```diff
@@ -42,8 +41,10 @@ type SchemeFunc func(*runtime.Scheme) error
 // ProvideFunc adds a controller to a Manager.
 type ProvideFunc func(manager.Manager) (controller.Controller, error)
 
-func main() {
-	flag.Parse()
+// crMain runs controllers written for controller-runtime. It's intended to be
+// called from main(). As controllers are migrated to use controller-runtime,
+// their initialization should be moved to this function.
+func crMain() error {
```

Comment:
I am not in-love with `crMain`, Perhaps `controllerRuntimeStart` or something more descriptive without `main`

The other thing I have seen is `_main` to be the thing called after some setup.

Reply:
:+1: I like `controllerRuntimeStart`. Renamed.